xbmc/filesystem/CurlFile.h

```diff
@@ -185,6 +185,7 @@ namespace XFILE
       bool m_allowRetry;
       bool m_verifyPeer = true;
       bool m_failOnError = true;
+      curl_slist* m_dnsCacheList;
```

Comment:
= nullptr

Then, there is no need to initialize in the ctor(s).

Reply:
fixed